python/tk_multi_workfiles/work_files.py

```diff
@@ -1050,3 +1058,66 @@ def _get_app_settings_for_context(self, context):
             for settings in engine_settings:
                 if settings.get("app_instance") == app_instance_name:
                     return settings.get("settings")
+
+
+def check_is_rez_compatible(new_ctx):
+    """verify that the given ctx is "compatible" with the current resolved rez environment.
+    Effectively: one shot/entity may have different packages version configured than another one..
+    :param new_ctx: The newly selected context.
+    :return: True if we either are not under Rez, or the new context is ok with the current one.
+    Else: Returns False.
+    """
+    rez_used_request = os.getenv('REZ_USED_REQUEST')
+    if not rez_used_request:
+        # not under Rez so it's always "ok"..
+        return True
+
+    sorted_prev_rez_used_request = sorted(rez_used_request.split())
+
+    # get the "rodeo facility" project :
+    rodeo_facility = new_ctx.sgtk.shotgun.find_one(
+        'CustomNonProjectEntity02', [['code', 'is', 'All rodeo']], fields=['code'])
```

Comment:
`All rodeo` it is actually defined in the config file used by rdo_rez_launcher...

but I agree it could be made more constants because I don't see why we would change it..  (it could break lot of stuffs I can guess).

hmm.. ok let's put it as const in our rez launcher..